lwtime/rfc1123z.go

```diff
@@ -0,0 +1,20 @@
+package lwtime
+
+import (
+	"strings"
+	"time"
+)
+
+type RFC1123Z struct {
+	time.Time
+}
+
+func (rfc *RFC1123Z) UnmarshalJSON(b []byte) (err error) {
+	t := strings.Trim(string(b), `"`)
+	rfc.Time, err = time.Parse(time.RFC1123Z, t)
+	return
+}
+
+func (rfc *RFC1123Z) MarshalJSON() ([]byte, error) {
```

Comment:
Could we return the stand time format for all Time types? We use https://datatracker.ietf.org/doc/html/rfc3339

Reply:
Fixed